docs/config.rst

```diff
@@ -726,8 +726,7 @@ Pip installer
 
    Determines the command used for installing packages into the virtual environment; both the package under test and its
    dependencies (defined with :ref:`deps`). Must contain the substitution key ``{packages}`` which will be replaced by
-   the package(s) to install.  You should also accept ``{opts}`` -- it will contain index server options such as
-   ``--pre`` (configured as ``pip_pre``).
+   the package(s) to install.  You should also accept ``{opts}`` -- it will contain index server options.
```

Comment:
As pre-release is configured now with the environment variable, I removed it from the example but do not have any idea what could be the replacement. 

As I understand, currently, all pip options could be configured using environment variables, so I'm not sure if `opts` is really required and maybe should be removed.